Classes/Command/IndexCommand.php

```diff
@@ -171,20 +170,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     protected function getSolrCores(int $pageId): array
     {
-        $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)
-            ->getQueryBuilderForTable('tx_dlf_solrcores');
-
         $solrCores = [];
-        $result = $queryBuilder
-            ->select('uid', 'index_name')
-            ->from('tx_dlf_solrcores')
-            ->where(
-                $queryBuilder->expr()->eq(
-                    'pid',
-                    $queryBuilder->createNamedParameter((int) $pageId, Connection::PARAM_INT)
-                )
-            )
-            ->execute();
+        $result = SolrCoreRepository::findByPid($pageId);
```

Comment:
That's not the way you access the repositories in Extbase usually. You know this guide? https://docs.typo3.org/m/typo3/book-extbasefluid/9.5/en-us/6-Persistence/2a-creating-the-repositories.html

Reply:
That's a temporar solution. I have read about it :) It is a first step in which I wanted to separate all places where actual access to database is performed and move it to repositories.